_schemaV2.graphql

```diff
@@ -2224,46 +2226,44 @@ type ArtistSeries {
     priceRange: String
     saleID: ID
     size: Int
+
+    # Filter results by Artwork sizes
     sizes: [ArtworkSizes]
     sort: String
     tagID: String
     width: String
   ): FilterArtworksConnection
   forSaleArtworksCount: Int!
+
+  # A globally unique ID.
+  id: ID!
   image: Image
-  imageHeight: Int
-  imageURL: String
-  imageWidth: Int
 
-  # Unique ID for this artist series
+  # A type-specific Gravity Mongo Document ID.
   internalID: ID!
   published: Boolean!
   representativeArtworkID: ID
   slug: String!
   title: String!
 }
 
-# The connection type for ArtistSeries.
+# A connection to a list of items.
 type ArtistSeriesConnection {
   # A list of edges.
   edges: [ArtistSeriesEdge]
-
-  # A list of nodes.
-  nodes: [ArtistSeries]
+  pageCursors: PageCursors!
 
   # Information to aid in pagination.
   pageInfo: PageInfo!
-
-  # Total count of matching nodes, before pagination
-  totalCount: Int!
+  totalCount: Int
```

Comment:
technically this got _weaker_ and could theoretically create an issue for a client that was using `totalCount` for `ArtistSeriesConnection` in an un-safe way -> haven't found any examples of that in clients though.

Reply:
Here's an example where `totalCount` _is_ being used - https://github.com/artsy/eigen/blob/83f9134e83195aae6bf6f732f5b13f171822124b/src/app/Scenes/ArtistSeries/ArtistSeriesMoreSeries.tsx#L40 - but in a way that should be safe even when it's typed as nullable